src/main/java/org/alliancegenome/curation_api/model/entities/ontology/OntologyTerm.java

```diff
@@ -22,6 +23,7 @@
 @Entity
 @Inheritance(strategy = InheritanceType.JOINED)
 @ToString(exclude = { "isaParents", "isaChildren", "isaAncestors", "isaDescendants", "crossReferences", "synonyms", "secondaryIdentifiers", "subsets"}, callSuper = true)
+@AGRCurationSchemaVersion("1.2.1")
```

Comment:
@mluypaert does this include 1.2.1 and all current pull requests? Because the "isaParents", "isaChildren", "isaAncestors", "isaDescendants" has not been released yet.

Reply:
Even though we have implemented it.

Reply:
Right, I missed that, these ancestry changes came in after merging in alpha into this branch so weren't represented in this schema version. I've now updated it to represent these new changes (now reported as `1.2.1+` like I did for other classes that require a new linkML model version which doesn't exist yet).